packages/meditrak-server/src/apiV2/index.js

```diff
@@ -3,7 +3,18 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
 
-import { catchAsyncErrors } from './middleware';
+import express from 'express';
```

Comment:
If you locally diff this file against the deleted addRoutesToApp, you'll notice very simple changes between the two. I was hoping github would detect it as a move and edit rather than a delete and create.